common/src/main/java/com/sequenceiq/cloudbreak/auth/crn/CrnGeneratorService.java

```diff
@@ -0,0 +1,77 @@
+package com.sequenceiq.cloudbreak.auth.crn;
+
+import java.util.UUID;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.auth.altus.Crn;
+import com.sequenceiq.cloudbreak.auth.altus.CrnResourceDescriptor;
+
+@Service
+@ConfigurationProperties("crn")
+public class CrnGeneratorService {
```

Comment:
How about RegionAwareCrnGenerator? I would like to convey what is the usecase for this service vs. the regular Crn.Builder.

Reply:
I like it, I will change the class name